src/plugins/template/backend/int_executable.cpp

```diff
@@ -130,21 +131,6 @@ bool runtime::interpreter::INTExecutable::call(const vector<shared_ptr<runtime::
             op_outputs.push_back(host_tensor);
         }
 
-        // get op type
```

Comment:
As i remember, we made such w/a (according specs) to get correct results for Subgraph tests. Please make sure that these changes do not broke ov functional tests

Reply:
In original implementation the `type` variable has been get from input but not used. For me it looks like remove not used code and should has no effect on functionality.

@artkuli Have you got any additional comment about this change?

Reply:
CI should show if we have any issues here.

Reply:
@ilyachur @artkuli @praasz 
I meant 2 types of functional tests:
1. opset conformance
2. plugin functional

Anyway, I will check the changes to compare results with previous version

Unresolve discussion before my verification.